### DISCUSSION TITLE
Update Old Sonoff 4CH V2

### DISCUSSION THREAD

**Discussion by Magiquall** [arendst/Tasmota#10655] [Q&A] [answered]
Hello,

I'm looking to make a home automation network for myself, and I have an old Sonoff 4CH V2, it still seems to work perfectly.
I had flashed it, at the time, with Tasmota 5.14 it seems to me, I just updated it respecting the steps up to 9.1.0.
But I saw that there were FR firmwares, I downloaded the one I found on this site: tasmota-FR.bin
But it doesn't work it crashes when I try and stays at version 9.1.0
So I don't dare take the tasmota32-FR.bin version.

As this is an old model it may have limitations.

Thank you

**Comment by barbudor** [comment, marked as answer]
Bonjour

tasmota-fr.bin should be exactly the same as tasmota.bin except with the french translations of all strings.
They should work the same exactly the same as the only differences are only text strings, not code.
You can go up to lastest release : v9.2.0
You can upgrade directly over the air if your sonoff has a proper access to the internet using the compressed binary `http://ota.tasmota.com/tasmota/release/tasmota-FR.bin.gz`

**Comment by ascillato** [comment]
Please, do not use tasmota32-FR.bin on a sonoff!

The 32 means for ESP32 devices only! As explained in the releases notes.

For a sonoff device or any device that uses an ESP8266 chip, you must use tasmota.bin or any other language variant.
